grace/models/classifier.py

```diff
@@ -26,14 +26,16 @@ def __init__(
         input_channels: int,
         hidden_channels: int,
         *,
-        output_classes: int = 2,
+        node_output_classes: int = 2,
+        edge_output_classes: int = 2,
     ):
         super(GCN, self).__init__()
         torch.manual_seed(12345)
```

Comment:
Is this needed here?

Reply:
Just incase it's not clear - I meant the seed

Reply:
Ah good spot! No, not needed I don't think.